src/devices/Common/Iot/Device/Common/CommonHelpers.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I can't remember if we discussed the fact of not having this project here and instead just have the *cs files and compile them in the projects we need them in, like the rest of Common. That said, I can address that in a separate PR to no longer block this one.